File: src/main/java/guru/xgm/vjass/psi/VjassTokenSets.java
package guru.xgm.vjass.psi;

import com.intellij.psi.tree.TokenSet;

import static guru.xgm.vjass.psi.VjassTypes.MULTI_LINE_COMMENT;
import static guru.xgm.vjass.psi.VjassTypes.SINGLE_LINE_COMMENT;

public interface VjassTokenSets {
    TokenSet COMMENTS = TokenSet.create(SINGLE_LINE_COMMENT, MULTI_LINE_COMMENT);
}

File: src/main/kotlin/raft/war/language/vjass/psi/VjassTokenSets.kt
package raft.war.language.vjass.psi

import com.intellij.psi.tree.TokenSet
import raft.war.language.vjass.psi.VjassTypes.BLOCK_COMMENT
import raft.war.language.vjass.psi.VjassTypes.LINE_COMMENT

interface VjassTokenSets {
    companion object {
        @JvmField
        val COMMENTS: TokenSet = TokenSet.create(LINE_COMMENT, BLOCK_COMMENT)
    }
}
